cpptod.dd

```diff
@@ -829,6 +831,124 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(LNAME2 functioninputparameters, Function Input Parameters)
+
+$(H4 The C++ Way)
+
+	$(B C++ does not support explicit input parameters.) If you use a
+	$(I pass-by-value) parameter $(D void function(int inputOnlyImplicitly)) it can just
+	be an input parameter implicitly. If you use a $(I pass-by-reference) parameter
+	usually $(D const) is applied to the parameter
+	$(D void foo(const int& inputOnlyParameterImplicitly)) that a forbidden
+	modification of the parameter in the function leads to a compilation error.
+
+$(CPPCODE
+void foo(int implicitInputParameterPassedByValue) {
+		// some code dependent on implicitInputParameterPassedByValue
+		...
+}
+
+void foo(int& implicitInputParameterPassedByReference) {
```

Comment:
Typically, starting curly braces go on the next line in D's documentation and source code.